python/ray/_private/services.py

```diff
@@ -447,7 +447,7 @@ def canonicalize_bootstrap_address(addr: str):
     if addr is None or addr == "auto":
         addr = get_ray_address_from_environment()
     try:
-        bootstrap_address = address_to_ip(addr)
+        bootstrap_address = convert_local_host_if_needed(addr)
```

Comment:
Can you update `ip:port` to `host:port` in the function comment?

Reply:
Sure.